examples/2-query/package.json

```diff
@@ -6,6 +6,7 @@
   "author": "Kara Stubbs <kara.stubbs@formidable.com>",
   "license": "MIT",
   "scripts": {
+    "build": "bsb -make-world",
```

Comment:
I'm happy to remove this if you'd like -- I kept accidentally trying to run `yarn build` and saw that the root project has a "build" script like this so thought it might be ok.

Reply:
No I think this is great! Sometimes I want to build one-off to check the compiler, other times I like to pass `-w` when hacking away quickly on the examples.